packages/web-app-files/src/components/Search/List.vue

```diff
@@ -242,24 +242,31 @@ export default defineComponent({
     }
 
     const buildSearchTerm = (manuallyUpdateFilterChip = false) => {
-      let term = ''
-      if (unref(searchTerm)) {
-        term = `+Name:*${unref(searchTerm)}*`
+      let query = ''
+      const add = (k: string, v: string) => {
+        query = (query + ` ${k}:${v}`).trimStart()
       }
 
-      const fullTextQuery = queryItemAsString(unref(fullTextParam))
-      if (fullTextQuery) {
-        term = `+Content:"${unref(searchTerm)}"`
+      const humanSearchTerm = unref(searchTerm)
+      const isContentOnlySearch = queryItemAsString(unref(fullTextParam)) == 'true'
+
+      if (isContentOnlySearch && !!humanSearchTerm) {
+        add('content', `"${humanSearchTerm}"`)
+      } else if (!!humanSearchTerm) {
+        add('name', `"*${humanSearchTerm}*"`)
       }
-      if (unref(scopeQuery) && unref(doUseScope) === 'true') {
-        term += ` scope:${unref(scopeQuery)}`
+
+      const humanScopeQuery = unref(scopeQuery)
+      const isScopedSearch = unref(doUseScope) === 'true'
+      if (isScopedSearch && humanScopeQuery) {
+        add('scope', `${humanScopeQuery}`)
       }
 
-      const tagsQuery = queryItemAsString(unref(tagParam))
-      if (tagsQuery) {
-        tagsQuery.split('+')?.forEach((tag) => {
-          term += ` +Tags:"${unref(tag)}"`
-        })
+      const humanTagsParams = queryItemAsString(unref(tagParam))
+      if (humanTagsParams) {
+        // in the legacy implementation there was the option to query by multiple tags, e.g. string.split("+")
```

Comment:
I don't think we need to comment legacy code removal in the source code, a comment in the pr would be sufficient

Reply:
^ Agreed. The reason it was coded this way is because initially, we wanted to enable filtering via multiple tags. But we need the conjunctive normal form for that to work.

Reply:
conjunctive normal form works now, we can add multi tag in a follow up pr. needs slight ui changes too.